src/allencell_ml_segmenter/main/segmenter_layer.py

```diff
@@ -24,8 +24,9 @@ class ImageLayer(SegmenterLayer):
     path: Optional[Path] = None
     data: Optional[np.ndarray] = None
     source: Source = Source(name="sourcename")
```

Comment:
I'm still struggling to understand `Source`
1. Why is source inheriting from `SegmenterLayer`--Source is not a Layer right? It's an attribute of a layer
2. Do we actually need to mimic the fact that a real napari layer has a source attribute? It seems to me that we are only using `Source` for `Source.path`, but we already have a `path` attribute on `ImageLayer`. Is there a meaningful distinction between `Source.path` and `path`?